SampleApp/RoutingViewController.swift

```diff
@@ -41,11 +41,23 @@ class RoutingViewController: UIViewController, RoutingSearchDelegate {
     }
   }
 
+  override func viewDidAppear(animated: Bool) {
+    super.viewDidAppear(animated)
+    if LocationManager.sharedManager.isAlwaysAuthorized() || LocationManager.sharedManager.isInUseAuthorized() {
+      LocationManager.sharedManager.startUpdatingLocation()
```

Comment:
For the routing example do we need constant location updates or would it suffice to request a single location when the request is made?